zzzz_modular_occulus/code/modules/core_implant/cruciform/rituals/base.dm

```diff
@@ -112,12 +142,163 @@ Let there be light! Makes you glow for 5 minutes at a time.
+	origin_tech = list(TECH_BIO = 3, TECH_COMBAT = 5, TECH_POWER = 5) // Tech because this is kind of a weird anomaly I guess???
+	var/uses_left = 10
+
+/obj/item/weapon/tool/knife/dagger/nt/energy/New()
+	..()
+	set_light(l_range = 2, l_power = 2, l_color = COLOR_YELLOW) //make it glow
+
+/obj/item/weapon/tool/knife/dagger/nt/energy/afterattack(obj/target, mob/living/carbon/human/user, proximity)
+	..()
+	if(proximity && target && istype(target,/mob/living)) //only works on adjacent mobs so you don't get burnt for clicking on a mouse 7 tiles away
+		user.get_active_hand_organ().take_damage(0,5) //0 brute 5 burn
+		to_chat(user, SPAN_DANGER("\The [src] scorches your hand as you land a hit with it!"))
+		if(uses_left-- <1)
+			visible_message(SPAN_DANGER("\The [src] disintegrates!"))
+			qdel(src)
+
+/obj/item/weapon/tool/knife/dagger/nt/energy/throw_impact(atom/hit_atom)
+	..()
+	for(var/mob/living/carbon/M in hear(3, get_turf(src))) //half the range of what the SoT does
+		var/obj/item/weapon/implant/core_implant/I = M.get_core_implant(/obj/item/weapon/implant/core_implant/cruciform)
+		if(I && I.active && I.wearer)
+			continue
+		bang(get_turf(src), M)
+
+	for(var/obj/effect/blob/B in hear(5,get_turf(src)))       		//Blob damage here, range reduced as well
+		var/damage = round(30/(get_dist(B,get_turf(src))+1))
+		B.health -= damage
+		B.update_icon()
+
+	new/obj/effect/sparks(loc)
+	new/obj/effect/effect/smoke/illumination(loc, brightness=15)
+	if(src)
+		visible_message(SPAN_DANGER("\The [src] conflagarates in a bright flash of light!."))
+		qdel(src)
+
+//Holy copypasta, batman! This is bad. Real bad. But whatever.
+
+/obj/item/weapon/tool/knife/dagger/nt/energy/proc/bang(var/turf/T , var/mob/living/carbon/M)					// Added a new proc called 'bang' that takes a location and a person to be banged.
+	to_chat(M, SPAN_DANGER(pick("HOLY LIGHT!", "GOD HAVE MERCY!", "HOLY HAVEN!", "YOU SEE THE LIGHT!")))								// Called during the loop that bangs people in lockers/containers and when banging
+	playsound(loc, 'sound/effects/bang.ogg', 50, 1, 5)		// people in normal view.  Could theroetically be called during other explosions.
+																// -- Polymorph
+
+//Checking for protections
+	var/eye_safety = 0
+	var/ear_safety = 0
+	if(iscarbon(M))
+		eye_safety = M.eyecheck()
+		if(ishuman(M))
+			if(istype(M:l_ear, /obj/item/clothing/ears/earmuffs) || istype(M:r_ear, /obj/item/clothing/ears/earmuffs))
+				ear_safety += 2
+			if(HULK in M.mutations)
+				ear_safety += 1
+			if(istype(M:head, /obj/item/clothing/head/armor/helmet))
+				ear_safety += 1
+
+//Flashing everyone
+	if(eye_safety < FLASH_PROTECTION_MODERATE)
+		if (M.HUDtech.Find("flash"))
+			flick("e_flash", M.HUDtech["flash"])
+		M.Stun(2)
+		M.Weaken(10)
+
+
+
+//Now applying sound
+	if((get_dist(M, T) <= 2 || loc == M.loc || loc == M))
+		if(ear_safety > 0)
+			M.Stun(2)
+			M.Weaken(1)
+		else
+			M.Stun(10)
+			M.Weaken(3)
```

Comment:
This is a *very* long stun

Reply:
Again, identical to the SoT's.

Reply:
SoT has a very long activation time. More than enough time to get ganked if you don't prep for it.